Add release changelog (#396)
elastic/ecctl: docs/release_notes/v1.0.1.adoc

[id="{p}-release-notes-v1.0.1"]
=== Elastic Cloud Control (ecctl) v1.0.1
++++
<titleabbrev>v1.0.1</titleabbrev>
++++

<<{p}-release-notes-v1.0.1-bug-fixes,Bug fixes>> | <<{p}-release-notes-v1.0.1-changelog,Changelog>>

> Welcome to the v1.0.1 release of {n}. This version brings a bug fix.

Reviewer: ```suggestion
{n} version 1.0.1 includes a bug fix.
```

> Elastic Cloud Control (ecctl) is Elastic’s CLI to manage Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).

Reviewer: ```suggestion
Elastic Cloud Control (ecctl) is the CLI for Elasticsearch Service (ESS) and Elastic Cloud Enterprise (ECE).
```
